plugins/org.locationtech.udig.catalog/src/org/locationtech/udig/catalog/ID.java

```diff
@@ -468,7 +468,7 @@ public boolean isWFS() {
      * @return true if ID refers to a database (ie is a jdbc url)
      */
     public boolean isJDBC() {
-        return id.startsWith("jdbc"); //$NON-NLS-1$
+        return id.startsWith("jdbc") || (url != null && url.getProtocol().contains("jdbc")) ; //$NON-NLS-1$ //$NON-NLS-2$
```

Comment:
@nprigour Can you describe a scenario if the id wouldnt start with jdbc while protocol could be jdbc? I'm not sure but if the first is true the second statement isn't required. If the first is false the second should either, shouldn't it?